lib/src/domain/usecase/list_usecase_i.dart

```diff
@@ -0,0 +1,5 @@
+import '../../core/util/either_string_t.dart';
+
+abstract class ListUseCaseI<T> {
```

Comment:
Instead to ListUseCaseI you should call it IUseCase, the List of the name is not necessary, what happened in the future instead to use a list you use a map, Would this method make sense in that scenario?

Reply:
fixed